backend/models/Tweet.js

```diff
@@ -1,17 +1,63 @@
 const mongoose = require('mongoose');
 const Schema = mongoose.Schema;
 
-const tweetSchema = new Schema({
+// const tweetSchema = new Schema({
+//     author: {
+//         type: Schema.Types.ObjectId,
+//         ref: 'User'
+//     },
+//     text: {
+//         type: String,
+//         required: true
+//     }
+// }, {
+//     timestamps: true
+// });
+
+// module.exports = mongoose.model("Tweet", tweetSchema);
+
+const slideSchema = new Schema({
+    title: {
+      type: String,
+      required: true
+    },
+    content: {
+      type: String,
+      required: true
+    },
+    image: {
+      type: String
+    }
+    // videoUrl: {
+    //     type: String
+    // }
+  });
+  
+  const presentationSchema = new Schema({
+    title: {
+      type: String,
+      required: true
+    },
+    slides: [slideSchema],
```

Comment:
slides: {object}  
slides will have a page breakdown for each presentation

Reply:
sample state: 
Schema({
   
    slides: {
   (page) 1: { (page object) 1: {id: 1, startLeft=150, startTop = 100, text='sample'}),
    2: { 1: {id: 1, startLeft = 100, startTop = 50, text='page 2'}}
}